packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -184,10 +182,12 @@ class SwiftPackageManagerIntegrationMigration extends ProjectMigrator {
         '  ${_platform.name}/Runner.xcodeproj/project.pbxproj\n'
         '  ${_platform.name}/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme '
         '(or the scheme for the flavor used)\n\n'
-        'To avoid this failure, disable Flutter Swift Package Manager integration for the project\n'
+        'To add Swift Package Manager integration manually, please use the following instructions:\n'
+        'https://docs.flutter.dev/packages-and-plugins/swift-package-manager/for-app-developers#add-to-a-flutter-app-manually\n\n'
+        'Alternatively, to avoid this failure, disable Flutter Swift Package Manager integration for the project\n'
```

Comment:
Should we ask the user to send us a bug report by linking to https://github.com/flutter/flutter/issues/new/choose?

This might be noisy, but it'd be good for us to be aware of the reasons why the migration fails.

Reply:
We do, if you look at the complete error message, it's at the beginning.